cmd/gnodev/util_match.go

```diff
@@ -101,15 +107,20 @@ func splitRegexp(s string) filterMatch {
 				a = append(a, s[:i])
 				s = s[i+1:]
 				i = 0
+
 				continue
 			}
 		case '|':
 			if cs == 0 && cp == 0 {
 				a = append(a, s[:i])
+
```

Comment:
Can you also remove the one that enforces this please?

Reply:
This was part of the `wsl` linter

Reverted it here:

[6184454](https://github.com/gnolang/gno/pull/472/commits/6184454c69f70544466785372562e8dfba6e501f)